pkg/webhook/webhook.go

```diff
@@ -154,6 +154,13 @@ func deserializePod(ar *v1beta1.AdmissionReview) (corev1.Pod, error) {
 	if pod.ObjectMeta.Namespace != "" {
 		return pod, err
 	}
+
+	// AdmissionRequest contains an optional Namespace field
+	if ar.Request.Namespace != "" {
```

Comment:
any idea how AdmissionRequest namespace field is derived ?

Reply:
I think it is from webhook server where it knows the k8s object and populates the namespace when sending the AdmissionReview, however I didn't find the exact code the write this field.

Reply:
@zshi-redhat ok, sure, I thought it would use namespace configured from [here](https://github.com/k8snetworkplumbingwg/network-resources-injector/blob/master/deployments/webhook.yaml#L6) :)

Reply:
okay, it will not :-)
I tried not specifying namespace when creating pod and the AdmissionReview.Request.Namespace was assigned with right namespace by webhook server..